Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Using the official python 2.7 as a Base Image
+FROM python:2
+
+ADD . /edx-e2e-tests
+WORKDIR /edx-e2e-tests
+
+# Configuration
+RUN apt-get update
```

Comment:
A tip for reducing the size of Docker containers is to combine similar commands into a single RUN command. So, for example, you could combine both of the apt-get installs into one.